internal/locate/region_cache.go

```diff
@@ -1906,6 +1912,56 @@ func (c *RegionCache) UpdateBucketsIfNeeded(regionID RegionVerID, latestBucketsV
 	}
 }
 
+const cleanCacheInterval = 100 * time.Millisecond
```

Comment:
This seems to be a bit frequent for recycling memory, how about several seconds with a larger `cleanRegionNumPerRound`?

Reply:
It requires a lock to access the region cache. I hope the time that it holds the lock is short enough. So, I am very cautious about increasing `cleanRegionNumPerRound`.

I just tried `cleanRegionNumPerRound = 50`, the time of each round can be controlled under 20us. (But this does not involve removal). Maybe it's acceptable to change the value to 50 regions every 1 second.